osu.Game/Online/Notifications/Polling/PollingNotificationsClient.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Threading;
+using System.Threading.Tasks;
+using osu.Game.Online.API;
+
+namespace osu.Game.Online.Notifications.Polling
+{
+    /// <summary>
+    /// A notifications client which polls for new messages every second.
+    /// </summary>
+    public class PollingNotificationsClient : NotificationsClient
```

Comment:
I'd consider moving this to the test namespace (along with `PollingNotificationsClientConnector`) since it's only used there.